ImageNet/train/setu_resnet50_azure_ndv2.json

```diff
@@ -0,0 +1,40 @@
+{
+	"version": "v1.0",
+	"author": "Setu Chokshi",
+	"authorEmail": "setuc@hotmail.com",
+	"framework": "PyTorch 1.0",
+	"model": "ResNet50",
+	"hardware": "Azure ND40s_v2",
+  "timestamp": "2019-04-13",
+	"costPerHour": 12.24,
```

Comment:
Can you send me a link to the pricing information for an `ND40s_v2` instance from Azure?

Reply:
https://azure.microsoft.com/en-us/pricing/details/virtual-machines/linux/

Please scroll down to NDv2 Series preview. You can find the pricing there. These are currently available East US region.

Reply:
I fixed the filename, as per the required structure.